components/src/form-field/error-summary.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, Input } from "@angular/core";
+import { FormGroup } from "@angular/forms";
+
+@Component({
+  selector: "bit-error-summary",
+  template: ` <ng-container *ngIf="errorCount > 0">
+    <i class="bwi bwi-error"></i> {{ "fieldsNeedAttention" | i18n: errorString }}
+  </ng-container>`,
+  host: {
+    class: "tw-block tw-text-danger tw-mt-2",
+    "aria-live": "assertive",
+  },
+})
+export class BitErrorSummary {
+  @Input()
+  formGroup: FormGroup;
+
+  get errorCount(): number {
+    const errors = Object.values(this.formGroup.controls).filter((e) => e.touched && !e.valid);
```

Comment:
What about nested formGroups? I think you need to recurse, like this: https://github.com/bitwarden/web/blob/master/bitwarden_license/src/app/organizations/manage/sso.component.ts#L245 (although your implementation is easier to understand)

Reply:
Good catch. I copied your code.